themes/wim/components/wim/_block--menu-top-tasks.scss

```diff
@@ -24,35 +24,40 @@
         position: static;
         cursor: pointer;
         margin-bottom: 0;
-
-        @include MQ($screen-sm) {
-          &.open-default,
-          &:hover {
+        
+        &.open,
+        &.open-default {
+          .dropdown-menu {
+            display: block;
+          }
+          
+          .dropdown-toggle {
             background-color: $brand-primary;
-            .nolink {
+            color: #fff;
+
+            &:focus,
+            &:active,
+            &:hover {
+              background-color: $brand-primary;
               color: #fff;
             }
-            .dropdown-menu {
-              display: block;
-            }
-          }
-          &.open-default,
-          &:hover,
-          &.open {
-            .nolink {
+            @include MQ($screen-sm) {
               &:before {
                 display: block !important;
               }
             }
           }
         }
-        .nolink {
+        .dropdown-toggle {
           padding: 18px 10px;
           color: #000;
-          font-family: "Roboto Slab", "Droid Sans", Arial, sans-serif;
+          font-family: "Roboto Slab", "Droid Sans", Arial, sans-serif; //@todo use variable
           text-decoration: none;
           display: block;
           position: relative;
+          &:focus {
+            @extend %outline;
```

Comment:
For some components bootstrap removes the default outline. I made an extend for it so you can put the behaviour back in action. If you can give visual feedback in a better way for users that navigate with their keyboard that's also fine.